services/frontend-service/src/ui/components/ProductVersion/ProductVersion.tsx

```diff
@@ -0,0 +1,34 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright 2023 freiheit.com*/
+import './ProductVersion.scss';
+
+export type ProductVersionProps = {
+    environment: string;
+};
+export const ProductVersion: React.FC<ProductVersionProps> = (props) => {
+    const { environment } = props;
+    // TODO(GS): call tags endpoint
```

Comment:
either do the task behind the TODO, or add the story ID here ("will be done in story XYZ")